### PR TITLE
Standard

### LINKED ISSUE #87: Add standard

### PR DESCRIPTION
**Pull request by vweevers**
Closes #87.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,15 +27,16 @@
     "functional-red-black-tree": "^1.0.1",
     "immediate": "^3.2.3",
     "inherits": "~2.0.1",
-    "ltgt": "~2.2.0"
+    "ltgt": "~2.2.0",
+    "safe-buffer": "~5.1.1"
```
Comment (ralphtheninja) [inline package.json]: :+1:

**Comment by ralphtheninja** [comment]
@vweevers Wow, you're on a roll today! :)

**Comment by vweevers** [comment]
> Wow, you're on a roll today! :)

At home with the flu, nothing else to do ;)

**Comment by vweevers** [comment]
This is a big change so wouldn't mind another review.

**Comment by juliangruber** [comment]
> This is a big change so wouldn't mind another review.

did you run a command to achieve this?

**Comment by vweevers** [comment]
> did you run a command to achieve this?

`standard --fix`, but this only took care of ~10%. Did the rest manually.

**Comment by juliangruber** [comment]
What would you think about also running `prettier-standard '**/*.js'` before `standard --fix`, if this is generated 100% through tools it should be safer to merge without the laborious review.

**Comment by juliangruber** [comment]
and in case this doesn't get us to 100% yet, maybe split the commits between tool output and manual fixes.

**Comment by vweevers** [comment]
My git skills are gonna fail me so I'll do this in a new branch.

**Comment by juliangruber** [comment]
ok^^

**Comment by juliangruber** [comment]
the steps would have been something like this:

```bash
$ git checkout standard
$ git reset --hard master
$ # do all the things and commit
$ git push --force
```

**Comment by vweevers** [comment]
Yeah, but I'd maybe like to compare the end results of the two branches.

**Comment by juliangruber** [comment]
good point!

**Comment by vweevers** [comment]
Hmm, `prettier-standard` sadly doesn't fix [one-var](https://eslint.org/docs/rules/one-var). Also it's a little bit too aggressive in splitting long lines.

```js
  var keys = [], i, key
  for (i = 0; i < NUM_KEYS; i++) {
    keys.push(Math.random().toString(36).slice(2))
  }
```

becomes

```js
  var keys = [],
    i,
    key
  for (i = 0; i < NUM_KEYS; i++) {
    keys.push(
      Math.random()
        .toString(36)
        .slice(2)
    )
  }
```

**Comment by vweevers** [comment]
Also neither `standard --fix` or `prettier-standard` properly deal with ifs:

```diff
-    if (typeof this._start === 'undefined')
-      this._tree = tree.end
-    else if (ltgt.upperBoundInclusive(options))
-      this._tree = tree.le(this._start)
-    else
-      this._tree = tree.lt(this._start)
+    if (typeof this._start === 'undefined') this._tree = tree.end
+    else if (ltgt.upperBoundInclusive(options)) { this._tree = tree.le(this._start) } else this._tree = tree.lt(this._start)
```

**Comment by juliangruber** [comment]
wow, that output is terrible!

**Pull request closed without merging**